README.md

```diff
@@ -1 +1,269 @@
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<!-- > Describe how a new developer could make use of your project. -->
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Visual Studio Code
+- Live server Extension
+<!--
+Example command:
+
+sh
+ gem install rails
+
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+sh
+  cd my-folder
+  git clone https://github.com/ngalamac/Hello-Microverse.git
+
+
+### Install
+
+Install this project with:
+
+sh
+  cd my-folder
+  npm install
+
+
+
+### Usage
+
+To run the project, start the html file on live server
+
+### Run tests
+
+To run tests, run the following command:
+sh
+  npx hint .
+  npx stylelint "**/*.{css,scss}"
+
+
+### Deployment
+
+You can deploy this project using:
+
+sh
+  gh-pages
+  netlify
+  vercel
+
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 *Ngala Mac*
+
+- GitHub: [@ngalamac](https://github.com/ngalamac/Hello-Microverse.git)
+<!-- - Twitter: [@twitterhandle](https://twitter.com/twitterhandle) -->
+
+<!-- 👤 *Author2*
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- FUTURE FEATURES -->
+
+<!-- ## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] *[new_feature_1]*
+- [ ] *[new_feature_2]*
+- [ ] *[new_feature_3]*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<!-- Feel free to check the [issues page](../../issues/). -->
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- SUPPORT -->
+
+<!-- ## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project... -->
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<!-- > Give credit to everyone who inspired your codebase. -->
+
+I will like to thank microverse and all the students i've collaborated with
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- FAQ (optional) -->
+
+<!-- ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- *[Question_1]*
+
+  - [Answer_1]
+
+- *[Question_2]*
+
+  - [Answer_2] -->
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please 🙏🏻 add the valid `license` file with the name **MIT.md** file in your repo folder. It seems like you don't have this file and because of that the link is broken which makes your `README` file looks a little unprofessional and also may create an issue for someone who wants to contribute to your project. Kindly you can find the copy of the license [here](https://github.com/microverseinc/readme-template/blob/master/MIT.md). But don't ⏹️  forget to customize it with your name and project name.